gui/src/main/java/game/ScoreUtils.java

```diff
@@ -0,0 +1,134 @@
+package game;
+
+import java.util.Arrays;
+
+public class ScoreUtils {
+
+    private static int score;
+
+    static public int fullHouse(Dice[] dices) {
+        int[] counts = new int[6];
+        for (Dice dice : dices) {
```

Comment:
Hier wäre eine For-Schleife einfacher als diese For-Each-Schleife.

Reply:
Was macht das ganze schwieriger? Können wir es mit for each denn nicht lösen?

Reply:
Geht natürlich ist aber als wolle man mit Gabel und Messer Spagetti essen ;-)

Reply:
Okay ich verstehe. Löffel sind aber aktuell nicht abgewaschen. ;) Wir versuchen gerade aber neue zu organisieren.